### PR TITLE
Update admin.php

### PR DESCRIPTION
**Pull request by LyNac**
Fixes: https://github.com/pjhampton/Gramophone/issues/77

Linked admin logo to user's home page.

### CODE REVIEW

**Comment by pjhampton** [comment]
Nice @LyNac! This is awesome work! Can you please act on my review and I will get this merged as soon as I can?

File: config/admin.php [hunk 1]
```diff
@@ -74,6 +74,41 @@ function clean_dashboard(){
...
 /**
  *******************************************************************************
+ * change logo link from wordpress.org to user site
+ *******************************************************************************
+ *
+ * function changes url of the admin logo
+ *
+**/
+
+function login_url () {  return home_url(); }
```
Comment (pjhampton) [inline config/admin.php]: Please don't put the function definitions on one line to keep it consistant with the rest of the codebase. This should read:
```
function login_url () {  
    return home_url(); 
}
```

Comment (LyNac) [reply config/admin.php]: Will do!

**Comment by LyNac** [comment]
Done!

**Comment by pjhampton** [comment]
Cool @LyNac! Can you please commit and push your changes to this pr? 😄

**Pull request merged**